### ISSUE TITLE
Clarify "free trial" quota and model-order

### ISSUE THREAD

**Issue by EmmaWebGH** [continuedev/continue#913]
### Validations

- [x] I believe this is a way to improve. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [x] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) that requests the same enhancement

### Problem

Hi

I've been playing with this extension on and off, but wanted to find out more about the "free trial" re: GPT4, Phind, Gemini.
How do we know when this is up, or exceeded?

I can't find anything here in issues, in the docs, or on the website.
Only this line, which is pretty vague:
`Once you are using Continue regularly though, you will need to add an OpenAI API key that has access to GPT-4 by following these steps:`

I was trying to figure something out today, and noticed suddenly suggestions were all wrong. Asked the model which model it was, and it responded with GPT3. 
If it's going to degrade model, what's the order of degradation?

If GPT4 quota was up, should it silently go to 3, or provide feedback to the user than it's moving the 3? There is no GPT3 in the settings/json. If it's using 3 I would expect to see that reflected in the UI.

Or should it degrade silently to the next model in queue, for example Phind (which it doesn't appear to be doing currently).

Finally, on any of the options for trial, how do we know how that quota is proceeding?

Basically looking for more detailed information on 
a) how this may compare to something like Cody that has a set number of completions inc per month on the hobby/free tier
and
b) how we know "for sure" what model is being used in Continue since it appears to degrade/change silently with no visual/message cue.

Thanks

**Comment by sestinj** [comment]
@EmmaWebGH We don't ever degrade the model (what you see is what you get). However gpt-4 is known to respond by saying that it is gpt-3 or "based on gpt-3", as you could validate in the OpenAI playground like below.

<img width="1242" alt="Screenshot 2024-03-02 at 9 29 49 AM" src="https://github.com/continuedev/continue/assets/33237525/d0aba646-31f7-4910-8256-2865ab368548">

The free trial gives you 250 free requests and there will be a progress bar that shows up once you get to 125. I just checked and found that there was a bug in displaying this progress bar, which is why you might not have seen it. This is getting fixed right away.

<img width="431" alt="Screenshot 2024-03-02 at 9 38 40 AM" src="https://github.com/continuedev/continue/assets/33237525/2ab5a4a0-4bf1-4096-b64d-67198e8337b9">

**Comment by EmmaWebGH** [comment]
Thanks for this reply, true I couldn't see a progress bar anywhere!

Amazingly I must not have done 250 requests yet, as I've never seen that "limit reached" box. Though I thought "surely I must have by now", as I've been playing with it on and off for about a month. :)

Thanks for explaining about GPT4 being confused about it's identity as well.
I wish ALL models would be more accurate about the name of their own file. Many times (even with local LLMs) I've tried to double check what model I'm working with by asking it, and they never know! Ha.
